src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.lock.json

```diff
@@ -275,6 +275,9 @@
       "Microsoft.NETCore.ConsoleHost/1.0.0-beta-23328": {
         "type": "package"
```

Comment:
Sure, I'll remove it and add it to the .gitignore file so that it doesn't get picked up everytime that people build.